dnf-behave-tests/dnf/plugins-core/config-manager-addrepo.feature

```diff
@@ -0,0 +1,437 @@
+@dnf5
+Feature: dnf "config-manager" command - test "addrepo" subcommand
+
+
+Background:
+  Given I enable plugin "config_manager"
```

Comment:
This is not a problem of the PR, but personally I would prefer that we will not require to enable a particular plugin in test.

Reply:
Enabling of plugin was removed. Plugins are enabled by default in configuration file.